### DISCUSSION TITLE
Any planned updates for QuickLook compatibility? Annotations and Animation not working in ios AR

### DISCUSSION THREAD

**Discussion by rebrob** [google/model-viewer#4459] [Q&A] [answered]
Hello,

Currently, annotations and model animations are not working in ios AR. Are there any plans on model-viewer's part to update compatibility with QuickLook or is the hope that Apple embraces WebXR on all of their ios products?

Thanks for any insight or future roadmap plans.

Rebecca

**Comment by elalish** [comment, marked as answer]
WebXR support on iOS would certainly solve this. If anyone adds animation support to the three.js USDZExporter, then we'll get that support, but we don't have time to do it ourselves. As for annotations, that's not really possible with the way Quick Look is architected.

**Reply by rebrob** [reply]
Thanks for your quick reply.  We will hope that Apple embraces WebXR across their ios products, then.
